src/plugins/dataflow/components/ui/dataflow-program-topbar.scss

```
@@ -1,22 +1,24 @@
 @import "../../../../components/vars";
 @import "../dataflow-vars";
 
-$topbar-height: 44px;
+$topbar-height: 44px; //used to be 44
```

```suggestion
$topbar-height: 44px;
```
Comment doesn't make sense since the value hasn't changed.